src/main/java/blackjack/domain/game/GameAccount.java

```diff
@@ -0,0 +1,44 @@
+package blackjack.domain.game;
+
+import blackjack.domain.gamer.GameResult;
+import blackjack.domain.gamer.Player;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class GameAccount {
+
+    private static final Map<Player, Money> store = new LinkedHashMap<>();
+
+    public void betMoney(Player player, Money money) {
+        store.put(player, money);
+    }
+
+    public Money findMoney(Player player) {
+        return store.get(player);
+    }
+
+    public void applyGameResults(Map<Player, GameResult> gameResults) {
+        for (Player player : gameResults.keySet()) {
+            Money money = store.get(player);
+            GameResult gameResult = gameResults.get(player);
+            Money gameResultMoney = money.multipleRatio(gameResult.getRatio());
+            store.put(player, gameResultMoney);
+        }
+    }
+
+    public Money calculateDealerIncome() {
+        int dealerIncome = 0;
+        for (Money money : store.values()) {
+            dealerIncome += money.value();
+        }
+        return new Money(-dealerIncome);
+    }
```

Comment:
```suggestion
    public Money calculateDealerIncome() {
        int total = calculatePlayersTotalProfit()
        return new Money(-total);
    }
```

`플레이어 수익의 총합을 계산할 수 있다`라는 메서드로 구분할 수 있을 것 같네
책임 분리의 관점에서 고치는게 좋아보이는 것 같아 😀

그리고  이렇게 고치면서 가장 좋은 효과를 볼 수 있는 부분은 따로 있다고 생각하는데 바로 dealerIncome이라는 불변하지 않은 변수를 사용하지 않아도 된다는 것! 

상태가 변하게 되는 변수는 (맥락에 따라 다르겠지만) 시스템의 취약성과 직결된다고 생각해 
불변하게 사용할 수 있으면 사용하는게 개인적으로 좋다는 생각이야! 👊

Reply:
추가적으로 만약 `calculatePlayersTotalProfit`메서드가 `Money`를 반환하게 하고 

`Money`도메인에 `inverse`혹은 `negate`메서드 기능을 추가하면 금액을 마이너스로 바꾸는 책임을 책임에 맞는 도메인에 위임할 수 있을 듯 👍

Reply:
오오 리비 정말 좋은 의견인 것 같아
예리하군 👍 👍